docs/witness/user-guide.md

```diff
@@ -21,45 +21,45 @@
 
 ## For Users
 
-As a community member, to make an anonymous vote, choose the specific blockchain network and head over to the proposal of a particular project you're interested in voting for. 
+First, search for the project in the Search bar. Look for the proposal that you are interested in voting for.
```

Comment:
Before searching the project, user should choose the correct blockchain network listed on the tab, which can be found at the homepage